tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -189,6 +183,10 @@ For example, `13796619446086334065/tidb-0/tidb-audit-2022-04-21T18-16-29.529.log
 
 If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off**.
 
+> **Note:**
+>
+> Because the log file will be pushed to the cloud storage bucket when it is filled with 10M each time. Therefore, when the audit log is disabled, the log file less than 10M will not be automatically pushed to the cloud storage bucket. If you need to get this log file, you can contact [PingCAP support team](/tidb-cloud/tidb-cloud-support.md).
```

Comment:
```suggestion
> Each time the log file is filled with 10 MiB, it will be pushed to the cloud storage bucket. Therefore, after the audit log is disabled, the log file smaller than 10 MiB will not be automatically pushed to the cloud storage bucket. If you need to get this log file, contact [PingCAP support](/tidb-cloud/tidb-cloud-support.md).
```

Reply:
@TomShawn PTAL